bin/mympirun.py

```diff
@@ -42,81 +42,138 @@
 
 import sys
 import os
+import traceback
+import glob
+import inspect
 
 from vsc.utils import fancylogger
-from vsc.mympirun.mpi.factory import getinstance
-from vsc.mympirun.mpi.mpi import whatMPI
+from vsc.mympirun.rm.factory import getinstance
+import vsc.mympirun.mpi.mpi
```

Comment:
the `mpi.py` is the basic MPI class from the `vsc.mympirun.mpi` namespace. The other MPI flavors are also located in the `vsc,mympirun.mpi` namespace.